dev/devicelab/lib/tasks/hot_mode_tests.dart

```diff
@@ -23,7 +23,8 @@ TaskFunction createHotModeTest({ bool isPreviewDart2: true }) {
     final File benchmarkFile = file(path.join(_editedFlutterGalleryDir.path, 'hot_benchmark.json'));
     rm(benchmarkFile);
     final List<String> options = <String>[
-      '--hot', '-d', device.deviceId, '--benchmark', '--verbose', '--resident'
+      '--hot', '-d', device.deviceId, '--benchmark', '--verbose', '--resident',
+      '--local-engine', 'android_debug_unopt',
```

Comment:
Doesnt setLocalEngineOptionIfNecessary do this already?

Reply:
Ah, thanks for catching this. That was a change left from previous debugging experiments.